cmds/webboot/types.go

```diff
@@ -17,8 +19,9 @@ type ISO struct {
 	path  string
 }
 
+var _ = menu.Entry(&ISO{})
```

Comment:
Sorry I might've missed the discussion but why are these necessary?

Reply:
oh, so Ron suggest to use this code to replace all these "expected to implement ..." comment

Reply:
It's a very common idiom for ensuring that your type implements the interface.